ospf6d/ospf6_lsa.c

```diff
@@ -347,6 +347,7 @@ void ospf6_lsa_premature_aging(struct ospf6_lsa *lsa)
 	ospf6_flood_clear(lsa);
 
 	lsa->header->age = htons(OSPF_LSA_MAXAGE);
+	THREAD_OFF(lsa->expire);
```

Comment:
This event is already being cancelled a few lines above. Do we need to do this again here?

Reply:
missed it I will remove